gradio/components/chatbot.py

```diff
@@ -250,6 +259,30 @@ def __init__(
             ]
         self.placeholder = placeholder
 
+        self.suggestions = suggestions
+        if self.suggestions is not None:
+            for i, suggestion in enumerate(self.suggestions):
+                if "icon" in suggestion:
+                    suggestion["icon"] = self.serve_static_file(suggestion["icon"])
+                file_info = suggestion.get("files")
+                if file_info is not None and not isinstance(file_info, list):
+                    raise Error(
+                        "Data incompatible with files format. The 'files' passed should be a list of file paths or URLs."
+                    )
+                files = []
+                if file_info is not None:
+                    for x, file in enumerate(file_info):
+                        if file is not None:
```

Comment:
What's the use case for allowing `None` files? I don't think we should allow. Seems confusing.